src/runtime_src/core/tools/common/Process.cpp

```diff
@@ -37,6 +37,7 @@
 #include "ProgressBar.h"
 #include "Process.h"
 #include "EscapeCodes.h"
+#include "XBUtilitiesCore.h"

```

Comment:
I like it that you initially added XBUtilitiesCore.h prior to XBUtilities.h.  Doing so helps greatly in verifying that there are no longer any dependencies between these two files.

Reply:
agreed.